packages/connect-voting-disputable/src/thegraph/parsers/collateralRequirements.ts

```diff
@@ -17,7 +17,7 @@ export function parseCollateralRequirement(
     {
       id: collateralRequirement.id,
       voteId: collateralRequirement.vote.id,
-      token: collateralRequirement.token.id,
+      tokenId: collateralRequirement.token.id,
```

Comment:
Should we update anything else related to this change? :thinking:

Reply:
I double-check and no we don't. It was a change introduced during a merge conflict.